eng/common/config/labels.ts

```diff
@@ -49,11 +49,15 @@ export const AreaLabels = defineLabels({
   },
   "emitter:client:csharp": {
     color: "e1b300",
-    description: "Issue for the C# client emitter: @typespec/emitter-client-csharp",
+    description: "Issue for the C# client emitter: @typespec/http-client-csharp",
   },
   "emitter:client:java": {
     color: "e1b300",
-    description: "Issue for the Java client emitter: @typespec/emitter-client-java",
+    description: "Issue for the Java client emitter: @typespec/http-client-java",
+  },
+  "emitter:client:python": {
+    color: "ffa3f8",
```

Comment:
is there a req for our color to be the same as csharp and java?

Reply:
for now yeah we have 1 color per category of emitters